src/lib/select/select.ts

```diff
@@ -415,9 +415,16 @@ export class MatSelect extends _MatSelectMixinBase implements AfterContentInit,
   /** Input that can be used to specify the `aria-labelledby` attribute. */
   @Input('aria-labelledby') ariaLabelledby: string;
 
-  /** An object used to control when error messages are shown. */
+  /** Object used to control when error messages are shown. */
   @Input() errorStateMatcher: ErrorStateMatcher;
 
+  /**
+   * Function used to sort the values inside a select in multiple mode.
+   * Follows the same logic as `Array.prototype.sort`.
+   */
+  @Input() sortPredicate: (a: MatOption, b: MatOption, options: MatOption[]) => number =
```

Comment:
"Predicate" isn't the right term, since a predicate is something that always returns a boolean value. I think `sortComparator` would be more accurate

Reply:
TIL. I've renamed it to `sortComparator`.